installer/home/pom.xml

```diff
@@ -110,5 +110,11 @@
             <version>${project.version}</version>
             <type>pom</type>
         </dependency>
+        <dependency>
+            <groupId>org.vivoweb</groupId>
+            <artifactId>vitro-languages-home-core</artifactId>
+            <version>${project.version}</version>
+            <type>tar.gz</type>
```

Comment:
Out of curiosity, why is vivo-languages type pom and vitro-languages tar.gz?

Reply:
I think you are right, my thought it that as we publish [poms](https://oss.sonatype.org/content/repositories/snapshots/org/vivoweb/vivo-languages-core/1.12.3-SNAPSHOT/)  it should be set to pom to be downloaded.